userguide/tutorials/metrics-cloudwatch.adoc

```diff
@@ -60,8 +60,20 @@ AWS X-Ray is a service that collects and analyzes data from your applications. A
 To setup the daemon, log in to your EC2 instance and run the following commands:
 
 ```
-curl https://s3.dualstack.us-east-1.amazonaws.com/aws-xray-assets.us-east-1/xray-daemon/aws-xray-daemon-3.x.deb -o /home/ubuntu/xray.deb
-sudo dpkg -i /home/ubuntu/xray.deb
+# Download the X-Ray daemon package
+wget https://s3.us-east-2.amazonaws.com/aws-xray-assets.us-east-2/xray-daemon/aws-xray-daemon-3.x.deb
```

Comment:
Would it make sense to include this in our AMI - I thought we had discussed this already?